bugs/lp-1812676.txt

```diff
@@ -4,12 +4,40 @@ https://bugs.launchpad.net/ubuntu/+source/cloud-init/+bug/1812676
 === Begin SRU Template ===
 [Impact]
 
+Using cloud-init-per with a dashed name will result in the specified command
+being run more often than expected.
+
 [Test Case]
 
+```
+cat > lp-1812676.yaml << EOF
+#cloud-config
+bootcmd:
+- cloud-init-per instance mycmd-bootcmd touch /root/lp-1812676
+EOF
```

Comment:
One of the scenarios mentioned was if something else called 'cloud-init-per' (not just cloud-init);  The bug was in this outside caller since the stand-alone shell script didn't name the semaphore the same.  I think we need to include invoking cloud-init-per outside cloud-init here to confirm that it doesn't run a second time (it ran once via bootcmd) cloud-init-per should not re-run it.